ingestion-data/collections/CMIP245-winter-median-pr.json

```diff
@@ -39,5 +41,21 @@
     },
     "dashboard:is_periodic": false,
     "dashboard:time_density": "year",
-    "stac_version": "1.0.0"
+    "stac_version": "1.0.0",
+    "renders": {
+        "dashboard": {
+            "resampling": "bilinear",
+            "bidx": 1,
```

Comment:
FYI: right now (will be fixed in next titiler/rio-tiler version) you cannot pass `bidx=1` to titiler STAC endpoint but you need to pass `asset_bidx=cog_default|1`

Reply:
ref https://github.com/developmentseed/titiler/pull/724

Reply:
Does it also need to be an array? Like this:
"asset_bidx": [1]

Reply:
@vincentsarago looks like `bidx=1` currently works with `preview` and `map` endpoints. Will that be changing in the future?

Reply:
it shouldn't change 👍